gef-ui/server/controllers/submit-to-ukef/index.test.js

```diff
@@ -30,6 +48,10 @@ const MockApplicationResponse = () => {
     createdAt: '1625482095783',
     comment: 'The client needs this asap.',
   }];
+  res.bankId = 'BANKID';
+  res.exporter = mockExporter;
+  res.eligibility = { status: 'Draft' };
+  res.facility = { status: 'Draft' };
```

Comment:
I don't think need eligibility/facility/exporter here now 

if so I think 'Draft' should be 'IN_PROGRESS'

Reply:
fails test if removed

Reply:
removed